packages/babel-plugin-jsx/src/transform-vue-jsx.ts

```diff
@@ -86,6 +86,15 @@ const buildProps = (path: NodePath<t.JSXElement>, state: State) => {
 
         const attributeValue = getJSXAttributeValue(prop, state);
 
+        if (name === 'class' && attributeValue?.type === 'StringLiteral') {
```

Comment:
Not only for `class`, but for all properties.
https://babeljs.io/repl#?browsers=&build=&builtIns=false&corejs=3.21&spec=false&loose=false&code_lz=MYewdgzgLgBAggBwTAvDAFASlQPgwKBhgB4ATASwDdCiYoBTaFAIhtpgDMQQ3aAjAIYAnXqyI5eACXoAbGSBgB1EEJmkaxAPQVKEzPiA&debug=false&forceAllTransforms=false&modules=false&shippedProposals=false&circleciRepo=&evaluate=false&fileSize=false&timeTravel=false&sourceType=module&lineWrap=false&presets=react&prettier=false&targets=&version=7.18.1&externalPlugins=&assumptions=%7B%7D

Reply:
@sxzz, fixed

Reply:
You just reverted the changes you made before.

Reply:
My bad. Fixed. Again.